src/space.js

```diff
@@ -57,7 +57,12 @@ export default class ColorSpace {
 		this.referred = options.referred;
 
 		// Compute ancestors and store them, since they will never change
-		this.#path = this.#getPath().reverse();
+		Object.defineProperty(this, "path", {
```

Comment:
should this have some `@private` jsdoc to let IDEs know they should treat it as private ?

Reply:
It’s not private, just immutable. Private properties were used  in conjunction with a public getter to make it immutable.

Reply:
It doesn’t really need to be private, as long as it’s immutable.